terraso_backend/apps/graphql/schema/commons.py

```diff
@@ -25,7 +26,25 @@ def resolve_total_count(self, info, **kwargs):
         return self.length
 
 
-class BaseWriteMutation(relay.ClientIDMutation):
+class BaseMutation(relay.ClientIDMutation):
+    class Meta:
+        abstract = True
+
+    errors = GenericScalar()
+
+    @classmethod
+    def mutate(cls, root, info, input):
+        try:
+            return super().mutate(root, info, input)
+        except Exception as error:
+            logger.error(
```

Comment:
question: do we want to use `logger.exception`?

Reply:
@paulschreiber I think that is a good idea to get the stacktrace and know the specific mutation easier